src/main/java/uk/gov/ons/census/casesvc/config/QueueSetterUpper.java

```diff
@@ -102,6 +108,11 @@ public Queue rhUacQueue() {
     return new Queue(rhUacQueue, true);
   }
 
+  @Bean
+  Queue surveyLaunchedQueue() {
```

Comment:
Should this be `public Queue surveyLaunchedQueue()`?

Reply:
Or doesn't it really matter because this file will be gone soon?

Reply:
Yes. Good spot.